daml-assistant/src/DAML/Assistant/Types.hs

```diff
@@ -68,6 +69,10 @@ data InstallOptions = InstallOptions
     , iQuiet :: QuietInstall
     } deriving (Eq, Show)
 
+newtype InstallURL = InstallURL
```

Comment:
Example of what an install url looks like please. Does it have .tar.gz at the end? .zip? Qualified? With https?

Reply:
:+1: